docs/root/faq/performance/how_to_benchmark_envoy.rst

```diff
@@ -0,0 +1,41 @@
+What are best practices for benchmarking Envoy?
```

Comment:
@oschaaf @jmarantz @mattklein123 @PiotrSikora please let me know if you have any additions or modifications you'd like to see in this FAQ entry. This is really a commonly asked question.

Reply:
@mum4k as well

Reply:
Thanks this looks amazing. The only thing I can think of to add is possibly disabling circuit breakers which is a common issue with the default 1024 limit. Maybe just a brief discussion and a link to https://www.envoyproxy.io/docs/envoy/latest/faq/load_balancing/disable_circuit_breaking?